testing/benchmark/README.md

```diff
@@ -74,10 +74,18 @@ The main commands are:
 
 Helper commands
 
-- `~/.ssh/id_rsa_terraform`: Generates a new SSH key without passphrase for the worker VMs.
+- `./id_rsa_terraform`: Generates a new SSH key without passphrase for the worker VMs.
```

Comment:
If we revert the value of SSH_KEY, we also need to update the help here.

Reply:
De didn't revert it. So it's `./id_rsa_terraform` now

Reply:
done